<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadDlg</class>
 <widget class="QDialog" name="UploadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="detailText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入详细情况</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addrLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>路况</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="accRadioBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>事故</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="jamRadioBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>拥堵</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="blockRadioBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>封路</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
